roles/postgresql/tasks/backup_db.yml

```diff
@@ -1,25 +1,27 @@
 ---
 # back up the database to a location on the app's host (not the db host)
-- name: Check postgres backup path
-  become: true
-  become_user: "{{ deploy_user }}"
-  ansible.builtin.file:
-    dest: "{{ db_backup_path | dirname }}"
-    state: directory
-    owner: "{{ deploy_user }}"
-    group: "{{ deploy_user }}"
-    mode: 0744
-
 - name: Backup postgres database
-  become: true
-  become_user: "{{ deploy_user }}"
-  community.postgresql.postgresql_db:
-    name: "{{ application_db_name }}"
-    login_host: "{{ postgres_host }}"
-    port: "{{ postgres_port }}"
-    login_user: "{{ application_dbuser_name }}"
-    login_password: "{{ application_dbuser_password }}"
-    encoding: "UTF-8"
-    owner: "{{ application_dbuser_name }}"
-    state: "dump"
-    target: "{{ db_backup_path }}"
+  tags: db_backup
```

Comment:
This needs to go under the `block` definition, if it applies to all tasks within the block.

Reply:
I tried that and it doesn't work - I get a syntax error. I think it's behaving as expected?

However, I'm glad I took another look - when I tested this I did figure out that my recommended syntax for calling the playbook didn't need to specify the skip tags, since we're already limiting to specific tags.